### ISSUE TITLE
Frio - Modal Editor: scheduled posts will be sent immediately

### ISSUE THREAD

**Issue by loma-one** [friendica/friendica#14454]
- [ x ] I have searched open and closed issues for duplicates

### Bug Description

Create a new post with the Modal Editor.
Send the post at a later date

Actual Result:

The contribution will be sent immediately

Expected Result:

The post will be saved as a scheduled post and sent at the specified time.

### Platform Info

Friendica Version: dev 1573/1573 Post-Update-Version 1550/1550.

Friendica Source: git

PHP version: 8.2

SQL version: 10.11

**Comment by haheute** [comment]
I have just tested this on my instance and it seemed to work. I have posted a picture and some text at 20 o'clock. CET Time (Berlin), and it did only appear under the tab "Scheduled posts" (Geplante Beiträge).
At 20:10 it appeared in the other timelines.. just as it should.

Is  your timezone correct? Maybe the system thinks, when you say: "post at 8 p.m." that it is already after that time.
But I'm only guessing.
